tests/integration/helpers.py

```diff
@@ -304,6 +308,7 @@ async def deploy_and_relate_application_with_postgresql(
         apps=[application_name],
         status="active",
         raise_on_blocked=False,  # Application that needs a relation is blocked initially.
+        raise_on_error=False,
```

Comment:
This is to avoid transient errors after deploy for charms that we relate to postgres using this helper function. This helped avoid some of the errors in `mattermost` charm on TLS test, like I commented today.